CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx

```diff
@@ -46,6 +46,7 @@ export default function CreateReport({ onDone }) {
       // Setting form data to year and cdcFile
       const formdata = new FormData()
       formdata.append("cdc_file", cdcFile)
+      formdata.append("isCDCFilter", isCDCFilter)
```

Comment:
instead of passing the isCDCFilter in the formdata, pass it in via the query parameters like how year is done on line 53.